src/bot/bot.py

```diff
@@ -16,12 +16,13 @@
     BOT_LAST_BLOCK,
     COLLATERALS_ZONES_PERCENT,
     COLLATERALS_ZONES_VALUE,
+    COLLATERALS_ZONES_CURRENCY,
     ETH_LATEST_BLOCK,
     FETCH_DURATION,
     PROCESSING_COMPLETED,
 )
 from .parsers import OnChainParser
-
+from .middleware import wstethLastPrice, ethLastPrice, stETHLastPrice
```

Comment:
Should conform to `snake_case`, and `middleware` not the best place, it's better to move to separate module in this case.